api/src/test/java/org/openmrs/module/eptsreports/reporting/mock/calculation/EptsCalculationsMock.java

```diff
@@ -0,0 +1,199 @@
+package org.openmrs.module.eptsreports.reporting.mock.calculation;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import mockit.Mock;
+import mockit.MockUp;
+import org.openmrs.Concept;
+import org.openmrs.Encounter;
+import org.openmrs.EncounterType;
+import org.openmrs.Location;
+import org.openmrs.Obs;
+import org.openmrs.Patient;
+import org.openmrs.PatientProgram;
+import org.openmrs.Program;
+import org.openmrs.api.context.Context;
+import org.openmrs.calculation.patient.PatientCalculation;
+import org.openmrs.calculation.patient.PatientCalculationContext;
+import org.openmrs.calculation.result.CalculationResultMap;
+import org.openmrs.calculation.result.ListResult;
+import org.openmrs.calculation.result.SimpleResult;
+import org.openmrs.module.eptsreports.reporting.calculation.EptsCalculations;
+import org.openmrs.module.reporting.common.TimeQualifier;
+
+/** Mocks {@link EptsCalculations} with org.jmockit */
+public class EptsCalculationsMock extends MockUp<EptsCalculations> {
```

Comment:
The only test class that depends on this is OnArtForMoreThanXmonthsCalcultionTest, so I am looking into how we can update calculationTest.xml to remove the need for this mock class.